draft-ietf-scitt-architecture.md

```diff
@@ -388,32 +388,36 @@ Registration Policies refer to additional checks over and above the Mandatory Re
 
 Transparency Services MUST maintain Registration Policies which govern whether or not a given Signed Statement is eligible for registration.
 
-Registration Policies MUST be made transparent and available to all clients of the Transparency Service by registering them as Signed Statements on the Append-only Log.
+Transparency Services MUST also maintain a list of trust anchors used to authenticate Issuers, which MAY be included in the registration policy statement.
+For instance, a trust anchor could be an X.509 root certificate, the discovery URL of an OpenID Connect identity provider, a trusted DID resolver, a GPG keyserver, or any other PKI trust anchor.
 
+Registration Policies and trust anchors MUST be made transparent and available to all clients of the Transparency Service by registering them as Signed Statements on the Append-only Log.
 
-
-This specification leaves implementation, encoding and documentation of Registration Policies to the operator of the Transparency Service.
+This specification leaves implementation, encoding and documentation of Registration Policies and trust anchors to the operator of the Transparency Service.
 
 #### Mandatory Registration Checks
 
-Transparency Services MUST, at a minimum, perform the following checks before registering a Signed Statement:
+During registration, a Transparency Services MUST, at a minimum, authenticate the Issuer of the Signed Statement by validating the COSE signature and checking the identity of the issuer against one of its configured trust anchors, using the `x5t`, `x5chain` and `kid` protected headers of the Signed Statement as hints.
+For instance, for X.509 signed claims, the Transparency Service must build and validate a complete certificate chain from the Issuer's certificate identified by `x5t`, to one of the root certificates most recently registered as a trust anchor of the Transparency Service.
+
+The Transparency Service MUST evaluate the registration policy that was most recently added to the Append-only Log for instance-specific registration checks.
```

Comment:
can we define "recent", is that a logical time or lamport time? 

What is an "instance-specific registration check" ?

Reply:
recent = latest added to the append only log, do you think this needs to be explicit?

"instance-specific registration check" just means "registration policy"

Reply:
If we need to be specific about ranges of time, lamport order is pretty rigid and also essentially up to the trust service. Can we do something like "Mininum Merge Delay"

```suggestion
If the transparency service subjects signed statements to registration policies at time of registration, it MUST declare the maximum delay from accepting the policy to it being applicable to future statements.
```